docs/Reference/Host-Configuration/API-Reference/SystemdCheck.md

```diff
@@ -0,0 +1,58 @@
+<!-- THIS FILE IS AUTOMATICALLY GENERATED BY DOCBUILDER, DO NOT EDIT MANUALLY! -->
+
+# SystemdCheck
+
+A check that can be run on the host to ensure systemd service(s) are in a successful state, as defined by `systemctl status` returning success.
+
+| Characteristic | Value    |
+| -------------- | -------- |
+| Type           | `object` |
+
+## Properties
+
+### `name` **<span>(required)</span>**
+
+Name of the check.
+
+| Characteristic | Value    |
+| -------------- | -------- |
+| Type           | `string` |
+
+### `runOn` **<span>(required)</span>**
+
+List of servicing types that the check should run on. Valid servicing types are CleanInstall and AbUpdate, if All is specified, the check will run for both CleanInstall and AbUpdate.
+
+| Characteristic | Value   |
+| -------------- | ------- |
+| Type           | `array` |
+
+- Items of the array must have the type:
+
+   | Characteristic | Value                                                 |
+   | -------------- | ----------------------------------------------------- |
+   | Type           | `ServicingTypeSelection`                              |
+   | Link           | [ServicingTypeSelection](./ServicingTypeSelection.md) |
+
+### `systemdServices` **<span>(required)</span>**
+
+List of systemd services that need to be in successful state.
+
+| Characteristic | Value   |
+| -------------- | ------- |
+| Type           | `array` |
+
+- Items of the array must have the type:
+
+   | Characteristic | Value    |
+   | -------------- | -------- |
+   | Type           | `string` |
+
+### `timeoutSeconds` **<span>(required)</span>**
```

Comment:
is this timeout global or per service?

Reply:
does 0 have any special meaning?

Reply:
its for this check, so for all services this check is configured to verify

Reply:
updated to show that timeout=0 means execute check exactly 1 time.